fala/templates/adviser/search.html

```diff
@@ -7,70 +7,88 @@
 {% import "macros/element.html" as Element %}
 {% import "macros/forms.html" as Form %}
 
-{% block search_form %}
-  <form action="" class="laa-fala__search" novalidate>
-    <div
-      class="govuk-form-group
-        {% if (form.errors and form.errors['__all__']) %}
-          govuk-form-group--error
-        {% endif %}
-        "
-      id="fala_questions">
+{% block content %}
+  <div class="govuk-grid-column-two-thirds">
+      <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
+      <h1 class="govuk-heading-xl"> Find a legal aid adviser or family mediator </h1>
+      <p class="govuk-body">You need to provide information about your finances to find out if you qualify for legal aid.</p>
+      <p class="govuk-body"><a class="govuk-link" href="https://www.gov.uk/check-legal-aid" target="_blank" rel="noopener">Check if you qualify for legal aid (opens in a new tab)</a></p>
```

Comment:
URL to `check-legal-aid` should be in `base.py` and set up as a variable which can be accessed by the template. 

This allows the URL to be used in other places in the template and if any changes are made, you only need to change it one location. 

base.py
`CHECK_LEGAL_AID_URL = "https://www.gov.uk/check-legal-aid"`

Jinja Template:
`<p class="govuk-body"><a class="govuk-link" href="{CHECK_LEGAL_AID_URL}" target="_blank" rel="noopener"></p>`

Reply:
@psweeting1 that doesn't work.

Reply:
should have been this - href="{{ CHECK_LEGAL_AID_URL }}"